src/imports/otp.js

```diff
@@ -91,16 +97,17 @@ export default new (class {
    * @param {string} code - code to be verified by cloudflare worker
    * @returns {Promise<object>}
    */
-  async checkOTP(mobile, code: string): Promise<object> {
+  async checkOTP(mobile, code: string, clientIp): Promise<object> {
```

Comment:
in otp.js we already have a method to extract the ip called extractIP()
1. use a single way to extract ip
2. have same method signature for checkOTP and sendOTP regarding where to get the client ip from